charts/nginx/templates/nginx-headers-configmap.yaml

```diff
@@ -11,8 +11,16 @@ metadata:
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
 data:
-  # Policy was generated using https://report-uri.com/home/generate
-  Content-Security-Policy: "default-src 'self' *.{{ .Values.global.baseDomain }}; script-src 'unsafe-inline' 'unsafe-eval' *.{{ .Values.global.baseDomain }} cdn.jsdelivr.net cdn.astronomer.io cdn.metarouter.io cdn.segment.com www.google-analytics.com js.stripe.com widget.intercom.io js.intercomcdn.com cdn.lr-ingest.io; img-src 'self' data: *; connect-src *.{{ .Values.global.baseDomain }} wss://*.{{ .Values.global.baseDomain }} e.metarouter.io api.segment.com api.segment.io api-iam.intercom.io wss://nexus-websocket-a.intercom.io https://grafana.com/blog/news.xml; style-src 'unsafe-inline' *.{{ .Values.global.baseDomain }} cdn.jsdelivr.net fonts.googleapis.com; frame-src js.stripe.com; font-src *.{{ .Values.global.baseDomain }} cdn.astronomer.io fonts.gstatic.com js.intercomcdn.com data:; worker-src blob:"
+  # https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy
+  Content-Security-Policy: >-
+    default-src 'self' *.{{ .Values.global.baseDomain }} ;
+    connect-src *.{{ .Values.global.baseDomain }} wss://*.{{ .Values.global.baseDomain }} ;
+    font-src *.{{ .Values.global.baseDomain }} cdn.astronomer.io fonts.gstatic.com data: ;
+    frame-src 'self' ;
+    img-src 'self' data: * ;
+    script-src 'unsafe-inline' 'unsafe-eval' *.{{ .Values.global.baseDomain }} cdn.astronomer.io ;
+    style-src 'unsafe-inline' *.{{ .Values.global.baseDomain }} fonts.googleapis.com ;
```

Comment:
I do not think we need `fonts.googleapis.com;`

Reply:
In case it helps anyone to review changes. Tried to match lines

<img width="1715" alt="image" src="https://github.com/user-attachments/assets/0c00dc93-be41-477a-b61d-728bb813abea">

Reply:
`worker-src blob` that was removed, does it affect the Houston worker?

Reply:
Thanks for posting that before and after! Here it is in text form:

### Before
```
Content-Security-Policy: >-
  default-src 'self' *.{{ .Values.global.baseDomain }} ;
  connect-src *.{{ .Values.global.baseDomain }} wss://*.{{ .Values.global.baseDomain }} e.metarouter.io api.segment.com api.segment.io api-iam.intercom.io wss://nexus-websocket-a.intercom.io https://grafana.com/blog/news.xml ;
  font-src *.{{ .Values.global.baseDomain }} cdn.astronomer.io fonts.gstatic.com js.intercomcdn.com data: ;
  frame-src js.stripe.com ;
  img-src 'self' data: * ;
  script-src 'unsafe-inline' 'unsafe-eval' *.{{ .Values.global.baseDomain }} cdn.jsdelivr.net cdn.astronomer.io cdn.metarouter.io cdn.segment.com www.google-analytics.com js.stripe.com widget.intercom.io js.intercomcdn.com cdn.lr-ingest.io ;
  style-src 'unsafe-inline' *.{{ .Values.global.baseDomain }} cdn.jsdelivr.net fonts.googleapis.com ;
  worker-src blob: ;
```

### After

```
Content-Security-Policy: >-
  default-src 'self' *.{{ .Values.global.baseDomain }} ;
  connect-src *.{{ .Values.global.baseDomain }} wss://*.{{ .Values.global.baseDomain }} ;
  font-src *.{{ .Values.global.baseDomain }} cdn.astronomer.io fonts.gstatic.com data: ;
  frame-src 'self' ;
  img-src 'self' data: * ;
  script-src 'unsafe-inline' 'unsafe-eval' *.{{ .Values.global.baseDomain }} cdn.astronomer.io ;
  style-src 'unsafe-inline' *.{{ .Values.global.baseDomain }} fonts.googleapis.com ;
  worker-src blob: ;
```

### word-diff

<img width="1739" alt="Screenshot 2024-11-05 at 1 42 31 PM" src="https://github.com/user-attachments/assets/3216f94c-fe5b-489e-935d-47eb6fd6f760">

Reply:
@Simpcyclassy `worker-src: blob` was not removed https://github.com/astronomer/astronomer/commit/7fed8843a918104226915a5846fba4780487234b#diff-c5e406fcf82441ef8e3d284b0f1a022f22911aa6fc9481957cd4a3872402a65cR23

Reply:
I see! I skipped the line when I was copying